### PR TITLE
enh(mc): refreshed the doc and added a notice about a known AIX bug

### PR DESCRIPTION
**Pull request by omercier**
## Description

- updated the procedure by generating it
- added a notice about the known AIX issue

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md [hunk 1]
```diff
@@ -139,98 +155,542 @@ Afin de superviser votre **AIX SNMP** en SNMP,  il est nécessaire de configurer
...
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
 ```
 
 La commande devrait retourner un message de sortie similaire à :
 
 ```bash
-OK: All CPUs usage are OK | 'cpu.utilization.percentage'=9000%;80;90;0;100 'core.cpu.utilization.percentage'=9000%;;;0;100 
+OK: All interfaces are ok | '*interface_name*#status'=;;;;'*interface_name*#interface.traffic.in.bitspersecond'=b/s;;;;'*interface_name*#interface.traffic.out.bitspersecond'=b/s;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md:417]: no values?

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md [hunk 2]
```diff
@@ -5,124 +5,139 @@ title: AIX SNMP
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+> Known issue with AIX version `07.02.0004` on CPU mode.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md:8]:
```suggestion
> A known issue with AIX version `07.02.0004` affects the CPU mode.
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md [hunk 3]
```diff
@@ -5,124 +5,139 @@ title: AIX SNMP
...
+> There is [a bug](https://www.ibm.com/support/pages/apar/IJ32354) on this version that causes the SNMP response related to CPU to report wrong values.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md:9]:
```suggestion
> There is [a bug](https://www.ibm.com/support/pages/apar/IJ32354) in this version that causes the SNMP response related to the CPU to report wrong values.
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md [hunk 4]
```diff
@@ -5,124 +5,139 @@ title: AIX SNMP
...
+> [The fix](https://www.ibm.com/support/pages/apar/IJ36641) is available starting version `07.02.0050`.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md:10]:
```suggestion
> [The fix](https://www.ibm.com/support/pages/apar/IJ36641) is available from version `07.02.0050`.
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md [hunk 5]
```diff
@@ -140,98 +155,542 @@ server. Please refer to the official documentation from IBM:
...
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
 ```
 
 The expected command output is shown below:
 
 ```bash
-OK: All CPUs usage are OK | 'cpu.utilization.percentage'=9000%;80;90;0;100 'core.cpu.utilization.percentage'=9000%;;;0;100 
+OK: All interfaces are ok | '*interface_name*#status'=;;;;'*interface_name*#interface.traffic.in.bitspersecond'=b/s;;;;'*interface_name*#interface.traffic.out.bitspersecond'=b/s;;;;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md:417]: no values?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md (hunk 1 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md]:
```suggestion
OK: All interfaces are ok | *interface_name*#interface.traffic.in.bitspersecond'=4212b/s;;;;' *interface_name*#interface.traffic.out.bitspersecond'=1242b/s;;;;
```

File: pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md (hunk 5 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/operatingsystems-aix-snmp.md]:
```suggestion
OK: All interfaces are ok | *interface_name*#interface.traffic.in.bitspersecond'=4212b/s;;;;' *interface_name*#interface.traffic.out.bitspersecond'=1242b/s;;;;
```

**Review by cg-tw — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**